packages/components/src/Chip/components/ChipSuffix/Chip.Suffix.test.tsx

```diff
@@ -31,4 +31,48 @@ describe("Chip Compound Components", () => {
     const elem = container.querySelector("." + styles.empty);
     expect(elem).toBeInTheDocument();
   });
+
+  it("renders suffix as button element when onClick is passed", () => {
+    const { container } = render(
+      <Chip.Suffix onClick={jest.fn()}>
+        <Icon name="cross" />
+      </Chip.Suffix>,
+    );
+
+    expect(container.querySelector("button")).toBeInTheDocument();
+  });
```

Comment:
Because we changed the logic (no longer rendering `button` or `span`), this test is no longer there. But I did take your suggestion and apply it to other/new tests in this PR.  `Chip.Suffix` will now only be a `span`, but with a possible refactor in the future to make this a sibling component, I think it's best to grab this element by `testID` in the event that the semantics of the suffix change in the future.